docs/admin/uninstall.md

```diff
@@ -2,10 +2,17 @@
 
 Here you will find any steps necessary to cleanly remove the App from your Nautobot environment.
 
-## Uninstall Guide
+## Database Cleanup
 
-Remove the configuration you added in `nautobot_config.py` from `PLUGINS` & `PLUGINS_CONFIG`.
+Prior to removing the plugin from the `nautobot_config.py`, run the following command to roll back any migration specific to this plugin.
 
-## Database Cleanup
+```shell
+nautobot-server migrate nautobot_plugin_data_validation_engine zero
+```
 
-Drop all tables from the plugin: `nautobot_plugin_data_validation_engine*`.
+!!! warning "Developer Note - Remove Me!"
```

Comment:
Fill or remove

Reply:
@DistantVoyager This is up to package maintainers pls.